src/components/calculator/PointsDisplay.tsx

```diff
@@ -25,16 +30,15 @@ function displayPercent(points: number): string {
 function tooManyPoints(points: number): boolean {
```

Comment:
All references to MAX_POINTS and tooManyPoints should be removed now that we report up to 1M

Reply:
Done.